pennylane/qnn/keras.py

```diff
@@ -330,3 +351,34 @@ def input_arg(self):
         `Layer <https://www.tensorflow.org/api_docs/python/tf/keras/layers/Layer>`__. Set to
         ``"inputs"``."""
         return self._input_arg
+
+    @staticmethod
+    def set_input_argument(input_name: Text = "inputs") -> None:
+        """
+        Set the name of the input argument.
+
+        Parameters
+        ----------
+        input_name (Text): Name of the input argument
+        """
+        KerasLayer._input_arg = input_name
+
+    def get_config(self) -> dict:
+        """
+        Get serialized layer configuration
+
+        Returns
+        -------
+        configuration (dict): layer configuration
```

Comment:
And similarly here:
```suggestion
        Returns:
            dict: layer configuration
```
Note that the return doesn't have a name, just a type!

Reply:
For more details on the PL docstring conventions, we have a developer guide here: https://pennylane.readthedocs.io/en/stable/development/guide/documentation.html#docstrings

Feel free to also ask directly on this PR 👍 Happy to help out where I can!